### ISSUE TITLE
Spot -> Spot Consolidation

### ISSUE THREAD

**Issue by matti** [kubernetes-sigs/karpenter#763]
### Tell us about your request

originally discussed in aws/karpenter#1091, but when the conversation started to be fruitful "aws locked as resolved and limited conversation to collaborators"

### Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?

using karpenter costs a lot of money after a scale up when nodes are left running behind

### Are you currently working around this issue?

various approached discussed in #

### Community Note

please don't close this and keep conversation going.

**Comment by jonathan-innis** [comment]
I'm pretty sure that this is closed-out now with #897! You can enable spot-to-spot consolidation on v0.34.0 of Karpenter in the AWS provider with the `SpotToSpotConsolidation` feature flag.

**Comment by calebAtIspot** [comment]
Relevant: https://karpenter.sh/v0.34/reference/settings/

```
Environment Variable | CLI Flag | Description
-- | -- | --
FEATURE_GATES | --feature-gates | Optional features can be enabled / disabled using feature gates. Current options are: Drift,SpotToSpotConsolidation (default = Drift=true,SpotToSpotConsolidation=false)
```

Also, there's a interesting caveat to this feature:
```
The spot consolidation would kick-in only when there are a minimum of 15 instanceTypes for the new NodeClaim to replace the current spot candidate
```

This seems unlikely to effect you, unless you heavily restrict the possible instance types.
